packages/make-styles/src/types.ts

```diff
@@ -70,19 +70,19 @@ type MakeStylesCSSPseudos = {
 //
 
 type MakeStylesCSSObjectCustomL1 = {
-  [Property: string]: MakeStylesCSSValue | undefined | MakeStylesCSSObjectCustomL2;
+  [Property: string]: MakeStylesCSSValue | undefined | MakeStylesStrictCSSObject | MakeStylesCSSObjectCustomL2;
```

Comment:
Can you please add a test case for this to `packages/make-styles/src/types.test.ts`? 🙏

Reply:
Yes, I wanted to spin this change as a different PR before this one went in with included tests so they could keep their separate concerns. I'll post the PR link here once I do.

Reply:
I've written this as a separate issue in #21213

Reply:
Actually, I closed #21213 as I realized that the error in the end was actually complaining about the use of `borderColor` because it should be using shorthands instead, it was just a very weird error that didn't actually point to this.